test/info.py

```diff
@@ -62,6 +62,14 @@
     'lpc11u35_archlink_if': 0x9013,
 }
 
+# Maps a target application filename (sans extension) to the ID of the board 
+# it is built to run on. This mapping is also used to display a board name 
+# in the test output. Thus, name the target applications after the  board to
+# avoid confusing results.
+#
+# One-to-many is fine. Many-to-one is fine as long as only one (or none) target 
+# application is found on disk per connected board ID
+#
```

Comment:
I guess what you're saying, though, is that for the mbed cases, the file name cannot be arbitrary--it must match what's on the mbed site. If so, then, I would indeed include that in the comment. I can add that or you can.